lesson14_task2.py

```diff
@@ -0,0 +1,32 @@
+"""
+Write a decorator that takes a list of stop words and replaces them with * inside the decorated function
+def stop_words(words: list):
+    pass
+@stop_words(['pepsi', 'BMW'])
+def create_slogan(name: str) -> str:
+    return f"{name} drinks pepsi in his brand new BMW!"
+assert create_slogan("Steve") == "Steve drinks * in his brand new *!"
+"""
+
+from typing import Callable, List
+
+
+def stop_words(words: List[str]) -> Callable:
```

Comment:
Very nice solution!
Couple things to mention:
1) in newer python versions you could use list instead of typing.List typehints
2) you could use functools.wraps to preserve decorated function's name and docs